RenderSystems/GL/include/OgreGLFrameBufferObject.h

```diff
@@ -59,6 +59,10 @@ namespace Ogre {
     private:
         GLFBOManager *mManager;
         GLSurfaceDesc mMultisampleColourBuffer;
+        // mMultisampleColourBuffer.buffer is either shared through caching, or owned,
+        // if owned, mOwnedMultisampleColourBuffer contains mMultisampleColourBuffer.buffer
+        // otherwise, mOwnedMultisampleColourBuffer == nullptr
+        std::unique_ptr<GLHardwarePixelBufferCommon> mOwnedMultisampleColourBuffer;
```

Comment:
move this definition to the `GLRenderTexture` parent

Reply:
To the base class `GLFrameBufferObjectCommon` (defined in GLRenderTexture.h/.cpp)?

Reply:
yes. sorry, mixed them up